homeassistant/components/netatmo/__init__.py

```diff
@@ -106,8 +105,8 @@ def setup(hass, config):
             DATA_WEBHOOK_URL] = hass.components.webhook.async_generate_url(
                 webhook_id)
         hass.components.webhook.async_register(
-            DOMAIN, 'Netatmo', webhook_id, handle_webhook)
-        NETATMO_AUTH.addwebhook(hass.data[DATA_WEBHOOK_URL])
+            DOMAIN, DATA_NETATMO, webhook_id, handle_webhook)
```

Comment:
The second argument is the name of the webhook. Should we really change that? I think it will be shown in the frontend when listing registered webhooks.